cdisc_rules_engine/dataset_builders/variables_metadata_dataset_builder.py

```diff
@@ -7,7 +7,7 @@ def build(self):
          Returns the variable metadata from a given file as a dataframe.
          The resulting dataframe has the following columns:
         variable_name
-        variable_order
```

Comment:
Why change this name? Are you sure people aren't already using it?

Reply:
I changed it to have a consistent name with the attribute in the define xml. And I see the name needs to match the `define_...` name. Yes, I checked that people aren't already using it.